static/src/js/components/ShapefileUploadModal/ShapefileUploadModal.scss

```diff
@@ -26,6 +26,10 @@
     margin-right: calc(bootstrap.$spacer / 2);
   }
 
+  .btn-close {
```

Comment:
I don't understand this change. this should be core functionality of react-bootstrap, what are we doing to break it?

Reply:
From what I could find, the bootstrap upgrade likely changed how the close button is positioned within the stacking context which caused this issue. Why it's only on this modal, I'm not sure.

Reply:
When we upgraded bootstrap, in the EDSCModal we changed from rendering our own close button to using the react-bootstrap `closeButton` prop. It looks like we already used that prop on ShapefileUploadModal before the upgrade.

react-bootstrap docs specify that `onHide` is called when the `closeButton` is clicked, so we are doing something that breaks that base functionality. The fix for that shouldn't be to override the library's styles, we should find what we are doing to break it and fix that.